messaging/src/main/java/org/jboss/as/messaging/TransportConfigOperationHandlers.java

```diff
@@ -145,6 +145,16 @@ static void processConnectors(final OperationContext context, final Configuratio
                 final ModelNode config = property.getValue();
                 final Map<String, Object> parameters = getParameters(context, config);
                 final String clazz = FACTORY_CLASS.resolveModelAttribute(context, config).asString();
+                if (NettyConnectorFactory.class.getName().equals(clazz)) {
```

Comment:
I see. I added this check because there is another class InVMConnectorFactory the user may set here. (although there is a in-vm-connector which implies it).